sycl/test/regression/common_macro_name_collisions.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %clangxx -fsycl -fsyntax-only -Xclang -verify %s
```

Comment:
I suggest we extend existing sycl/test/regression/check_simple_name_collisions.cpp, which aims to perform similar check.

Reply:
Good eye. I completely missed that one.

Reply:
Moved, thanks!